src/cpp.hh

```diff
@@ -21,225 +21,227 @@
 
 }}} */
 
-
 #ifndef SRC_CPP_HH_
 #define SRC_CPP_HH_
 
-#include <vector>
-#include <string>
 #include <list>
+#include <string>
+#include <vector>
 
-#include "operator.hh"
-
-#include "printer.hh"
 #include "codegen.hh"
-
-#include "table.hh"
-#include "symbol_fwd.hh"
+#include "operator.hh"
 #include "para_decl_fwd.hh"
+#include "printer.hh"
 #include "product.hh"
+#include "symbol_fwd.hh"
+#include "table.hh"
```

Comment:
same critique applies here as for `src/cpp.cc`. This PR is overly hard to understand as the many thousand lines of code that changed just because of code-style are very hard to differentiate from conceptional changes :-/